### PR TITLE
fix: make Procfile compatible with herokuish

### PR DESCRIPTION
**Pull request by ghost**
rel: https://github.com/gliderlabs/herokuish/issues/326#issuecomment-365117882

### CODE REVIEW

File: bin/heroku-web [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```
Comment (SuperSandro2000) [inline bin/heroku-web]:
```suggestion
#!/usr/bin/env bash
```

and you are missing a new line.

Comment (aisk) [reply bin/heroku-web]: Should we added a `set -e` below this line?

Comment (SuperSandro2000) [reply bin/heroku-web]: `set -eou pipefail` if not commands fail.

**Comment by SuperSandro2000** [comment]
bump

**Review by Gargron — APPROVED** [review]

**Pull request merged**